nucypher/characters/chaotic.py

```diff
@@ -229,9 +227,9 @@ def register():
                 with ThreadedSession(self.db_engine) as session:
 
                     existing = Recipient.query.filter_by(address=new_address).all()
-                    if existing:
+                    if len(existing) > self.MAX_INDIVIDUAL_REGISTRATIONS:
                         # Address already exists; Abort
-                        self.log.debug(f"{new_address} is already enrolled.")
+                        self.log.debug(f"{new_address} is already enrolled {self.MAX_INDIVIDUAL_REGISTRATIONS} times - Please use another address.")
```

Comment:
This is just debugging right? Did you mean to use `Please use another address.` in the response on the next line?

Reply:
Oh - Yeah whoops.

Reply:
:heavy_check_mark: